packages/node/src/index.ts

```diff
@@ -1,5 +1,6 @@
-import { foo } from './example'
-// TODO
-
-console.log('hello world')
-foo(123)
+export { load } from './app/load'
```

Comment:
Can't comment in the example files but those can probably be removed now :smile: